home/xinitrc

```diff
@@ -0,0 +1,4 @@
+xrandr --auto
+xrdb ~/.Xresources
+
+awesome
```

Comment:
What's the difference between this file and `home/xprofile` ?

Reply:
I should've documented it at the time:
- https://wiki.archlinux.org/index.php/xprofile
- https://wiki.archlinux.org/index.php/Xinit#xinitrc
I was having an issue with LightDM, xprofile is executed by LightDM, but I don't think xinitrc was being executed. I should have xinitrc execute xprofile. `startx` will execute xinitrc and not xprofile